ui/raidboss/data/04-sb/trial/lakshmi-ex.js

```diff
@@ -9,7 +9,7 @@ export default {
   timelineFile: 'lakshmi-ex.txt',
   timelineTriggers: [
     {
-      id: 'Lakshmi Path of Light',
+      id: 'Lakshmi EX Path of Light',
```

Comment:
I think the pattern here is usually to include Ex as a suffix to the word, so LakshmiEx (or LakEx I see sometimes?)
I don't care here, commenting to find out if there's a rationale for it or if it doesn't matter at all.

Reply:
I think the only use of `EX` so far is `SophiaEX` (no space).  All of the other extremes are some variant of `SomethingEx`.  I don't think there's a rationale, but maybe it'd make sense to be consistent? I'm not sure why Susano, Byakko, Rathalos are all shortened other than I just didn't want long ids, but there's not a more consistent rationale.

Reply:
Fair point on the casing. I don't think we need to worry about the shortened forms of the names, but I'll definitely change it over to `LakshmiEx`.